site/server/app/com/fortysevendeg/exercises/services/interpreters/interpreters.scala

```diff
@@ -130,20 +164,19 @@ trait TaskInstances {
 }
 
 trait IdInstances {
-  implicit def idApplicativeError(
-    implicit
-    I: Applicative[cats.Id]
-  ): ApplicativeError[cats.Id, Throwable] = new ApplicativeError[cats.Id, Throwable] {
+  implicit val idMonad: Monad[cats.Id] with MonadError[cats.Id, Throwable] = new Monad[cats.Id] with MonadError[cats.Id, Throwable] {
```

Comment:
Let's consistently use either `cats.Id` or `Id` throughout this file-- I'd only want to use both if we have to for some scope reason.